src/AlfaCountry.php

```diff
@@ -0,0 +1,296 @@
+    'SO' => '', // __( 'Somalia', 'woocommerce' ),
+    'ZA' => 'ZA', // __( 'South Africa', 'woocommerce' ),
+    'GS' => '', // __( 'South Georgia/Sandwich Islands', 'woocommerce' ),
+    'KR' => 'KR', // __( 'South Korea', 'woocommerce' ),
+    'SS' => '', // __( 'South Sudan', 'woocommerce' ),
+    'ES' => 'E', // __( 'Spain', 'woocommerce' ),
+    'LK' => 'CL', // __( 'Sri Lanka', 'woocommerce' ),
+    'SD' => 'SUD', // __( 'Sudan', 'woocommerce' ),
+    'SR' => '', // __( 'Suriname', 'woocommerce' ),
+    'SJ' => '', // __( 'Svalbard and Jan Mayen', 'woocommerce' ),
+    'SZ' => '', // __( 'Swaziland', 'woocommerce' ),
+    'SE' => 'S', // __( 'Sweden', 'woocommerce' ),
+    'CH' => 'CH', // __( 'Switzerland', 'woocommerce' ),
+    'SY' => 'SYR', // __( 'Syria', 'woocommerce' ),
+    'TW' => '', // __( 'Taiwan', 'woocommerce' ),
+    'TJ' => '', // __( 'Tajikistan', 'woocommerce' ),
+    'TZ' => 'EAT', // __( 'Tanzania', 'woocommerce' ),
+    'TH' => 'T', // __( 'Thailand', 'woocommerce' ),
+    'TL' => '', // __( 'Timor-Leste', 'woocommerce' ),
+    'TG' => 'RT', // __( 'Togo', 'woocommerce' ),
+    'TK' => '', // __( 'Tokelau', 'woocommerce' ),
+    'TO' => '', // __( 'Tonga', 'woocommerce' ),
+    'TT' => '', // __( 'Trinidad and Tobago', 'woocommerce' ),
+    'TN' => 'TN', // __( 'Tunisia', 'woocommerce' ),
+    'TR' => 'TR', // __( 'Turkey', 'woocommerce' ),
+    'TM' => '', // __( 'Turkmenistan', 'woocommerce' ),
+    'TC' => '', // __( 'Turks and Caicos Islands', 'woocommerce' ),
+    'TV' => '', // __( 'Tuvalu', 'woocommerce' ),
+    'UG' => '', // __( 'Uganda', 'woocommerce' ),
+    'UA' => 'UA', // __( 'Ukraine', 'woocommerce' ),
+    'AE' => 'UAE', // __( 'United Arab Emirates', 'woocommerce' ),
+    'GB' => 'GB', // __( 'United Kingdom (UK)', 'woocommerce' ),
+    'US' => 'USA', // __( 'United States (US)', 'woocommerce' ),
+    'UM' => '', // __( 'United States (US) Minor Outlying Islands', 'woocommerce' ),
+    'VI' => '', // __( 'United States (US) Virgin Islands', 'woocommerce' ),
+    'UY' => 'ROU', // __( 'Uruguay', 'woocommerce' ),
+    'UZ' => 'UZ', // __( 'Uzbekistan', 'woocommerce' ),
+    'VU' => '', // __( 'Vanuatu', 'woocommerce' ),
+    'VA' => '', // __( 'Vatican', 'woocommerce' ),
+    'VE' => '', // __( 'Venezuela', 'woocommerce' ),
+    'VN' => 'VN', // __( 'Vietnam', 'woocommerce' ),
+    'WF' => '', // __( 'Wallis and Futuna', 'woocommerce' ),
+    'EH' => '', // __( 'Western Sahara', 'woocommerce' ),
+    'WS' => '', // __( 'Samoa', 'woocommerce' ),
+    'YE' => '', // __( 'Yemen', 'woocommerce' ),
+    'ZM' => '', // __( 'Zambia', 'woocommerce' ),
+    'ZW' => 'ZW', // __( 'Zimbabwe', 'woocommerce' ),
+  ];
+
+  /**
+   * Returns the alfa country vehicle code for a given ISO country code.
+   *
+   * @param string $iso_code
+   *   The ISO country code to lookup.
+   *
+   * @return string
+   */
+  public static function toAlfa(string $iso_code) {
+    return static::MAP[$iso_code] ?? $iso_code;
```

Comment:
The Null Coalescing Operator returns an empty string for all country codes with empty mapping value.  The idea was to pass the ISO code instead.